record/record_test.go

```diff
@@ -314,10 +314,25 @@ func TestStaleWriter(t *testing.T) {
 	}
 }
 
-func writeRecords(records [][]byte) ([]byte, error) {
+type testRecords struct {
+	records [][]byte // The raw value of each record.
+	buf     []byte   // The serialized records form of all records.
+}
+
+// makeTestRecords generates test records of specified lengths. The raw
+// records are returned as *testRecords. The first record will consist of
```

Comment:
I'm not sure if "the raw records are returned as *testRecords" adds anything that you can't already tell from the return value types.